src/networks.json

```diff
@@ -1624,5 +1624,19 @@
     "explorer": "https://blockexplorer.avax.boba.network/",
     "start": 4832,
     "logo": "ipfs://QmP1iuEynzcghrukQLfw7zh7BJwG3dJSwyZprqD4YHHsFD"
+  },
+  "1701": {
+    "key": "1701",
+    "name": "Anytype EVM Chain",
+    "shortName": "AnytypeChain",
+    "chainId": 1701,
+    "network": "mainnet",
+    "multicall": "0x4542a07dE7f8A1cEB85aEcBFD0d5338e9886EF49",
+    "rpc": [
+      "https://geth.anytype.io"
```

Comment:
HI @AnthonyAkentiev I am getting CORS issue with this node, this node won't work on the front end.?

Reply:
@ChaituVR Please check again. I've set --http.corsdomain '*' for now. 
What domain should I ad? Is it**snapshot.org**?

Reply:
I tried again but it still doesn't work, would be great if you can add `snapshot.org` and `snapshot-networks.on.fleek.co` . is there any restriction based on the IP/rate limit to this node?

Reply:
@ChaituVR fixed. please try again
sorry

Reply:
@AnthonyAkentiev I still face the same issue. it works fine 

```JSON
"1701": {
    "key": "1701",
    "name": "Anytype EVM Chain",
    "shortName": "AnytypeChain",
    "chainId": 1701,
    "network": "mainnet",
    "multicall": "0x4542a07dE7f8A1cEB85aEcBFD0d5338e9886EF49",
    "rpc": [
      "https://geth.anytype.io"
    ],
    "explorer": "https://explorer.anytype.io",
    "start": 1126886,
    "logo": "ipfs://QmaARJiAQUn4Z6wG8GLEry3kTeBB3k6RfHzSZU9SPhBgcG"
  },
  ```

Reply:
@AnthonyAkentiev I still face the same issue. it works fine when i use postman, but fails in chrome browser

Reply:
@ChaituVR 
Weird!

```
curl https://geth.anytype.io/ -H "Host: [geth.anytype.io](http://geth.anytype.io/)" -X POST -H "Content-type: application/json" -v

Response:
< HTTP/1.1 200 OK
< Date: Mon, 24 Oct 2022 16:52:59 GMT
< Content-Type: application/json
< Content-Length: 0
< Connection: keep-alive
< Server: nginx/1.18.0 (Ubuntu)
< Access-Control-Allow-Origin: *
< X-Host: ip-172-31-35-84.eu-central-1.compute.internal
```

Could you please post which request fails?
You can copy it directly from the Chrome's developers console ("Networking" tab). Thank you.

Reply:
@ChaituVR thx for you help.
I've fixed everything, please try again.

here is the debug page with a button that sends the POST request -> https://evm.anytype.io/debug.html
you can click it to try

worked in the Chrome too